### PR TITLE
[RetrievalClient] Add Information on Operators that are responding

### PR DESCRIPTION
**Pull request by siddimore**
## Why are these changes needed?
1. Log Total Available Operators
2. Log ChunkStatus fro Operator

## Checks

- [ ] I've made sure the lint is passing in this PR.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: clients/retrieval_client.go [hunk 1]
```diff
@@ -31,6 +31,11 @@ type retrievalClient struct {
 	numConnections        int
 }
 
+type operatorFetchChunkStatusInfo struct {
+	Success             bool
```
Comment (mooselumph) [inline clients/retrieval_client.go:35]: I don't think the success argument is actually used, so it doesn't seem like this needs a custom type.

File: clients/retrieval_client.go [hunk 2]
```diff
@@ -157,5 +179,12 @@ func (r *retrievalClient) RetrieveBlob(
 		return nil, err
 	}
 
+	// Log Operator Fetch Chunk Status
+	for _, statusInfo := range operatorFetchChunkStatus {
+		if !statusInfo.Success {
+			r.logger.Warn("Failed to receive chunks from operator", statusInfo.IndexedOperatorInfo)
```
Comment (mooselumph) [inline clients/retrieval_client.go:185]: The logger has the following signature: 
`Warn(message, key1, value1, key2, value2,...)`,

so you are missing a key.

**Comment by mooselumph** [comment]
@siddimore, can you make the PR from your own fork of the repo instead of adding a new branch. This is the new OSS approach we are taking.

File: clients/retrieval_client.go (hunk 1 above)
Comment (siddimore) [reply clients/retrieval_client.go:35]: yes initially when i wrote i thought of tracking both retrievals that succeed from operators and that do not from operators....your comment makes sense i will remove this custom type

**Pull request closed without merging**